docs/quickstart.md

```diff
@@ -15,10 +15,11 @@ N.B.  Ensure the current user has permission to run the `docker` command without
 ## Installation
 This guide will install Armada on 3 local Kubernetes clusters; one server and two executor clusters. 
 
-Set the `ARMADA_VERSION` environment variable and clone this repository with the same version tag as you are installing. For example to install version `v0.0.5`:
+Set the `ARMADA_VERSION` environment variable and clone this repository with the same version tag as you are installing. For example to install version `v0.1.0`:
```

Comment:
shall we use 0.1.1?

Reply:
Yes, at time of writing there weren't any binaries published for v0.1.1. Tested now and it works 👍